### ISSUE TITLE
All wallets not displayed

### ISSUE THREAD

**Issue by wyssen** [DeFiCh/app#183]
Hello dear team
There are other posts about this issue, but with my post, I can show the steps I did.
I have the DeFi app since version 2.1.3 on Win10. It worked well for the last week or two. Since I updated to version 2.1.4, I had to reindex on every opening of the app. It took about 2-3 hours for syncing every time.
Once it was done with sync, everything worked fine and I reinvested into the LM every day.

Since today however, after syncing, I was not able to see my wallets anymore (the eventually showed up during the sync usually). It shows the "Create a new wallet" or "Restore wallet from mnemonic seed".
I tried to close and reopen the app the whole day (it reindexed every time). However, it does not change.
I thought maybe a downgrade to 2.1.3 works, but there is the same error.

When I check in CLI with "gettokenbalances", I see my correct balances. And I would even be able to reinvest the tokens into the LM.
However, I am insecure to do it, because I cannot see the wallets.

Here are the outputs of the commands:
 gettokenbalances
[
  "85.80574138@0",
  "0.34001142@1",
  "0.01887382@2",
  "33.51692851@4",
  "23.02359603@5"
]

listlabels
[
  "",
  "BTC_over_DFI",
  "DFI_Rewards",
  "DFI_Token",
  "ETH_over_DEFI"
]

Even the liquitity pools with my parts are showing correctly, but not the wallets.
So I thought maybe I can make them reappear by importing a wallet backup I did yesterday. This however did not work either. Still no wallets shown.

The only thing I did not try yet is to reset the wallet and to reimport it with the seed. I am kind of afraid to do that, because a reset would mean that I'd loose the current state where I can see the coins via the CLI (gettokenbalances).

What do you recommend doing?
Thanks in advance for you support an effort.
wyssen

**Comment by jdavidhuebner** [comment]
Hi Team, Hi Wyssen,

I have exactly the same problem. Had my wallet(s) visible on 2.13. Then I downloaded 2.14, backupped my wallet before, installed 2.14. And since then, I can't access my wallets under the wallets tab. It only shows "Create a new wallet" or "Restore wallet from mnemonic seed" just like for Wyssen. Also 2.14 took ages to sync for me, so I downgraded back to 2.13. Hoping that would solve my problem. But still not solved. Now I feel insecure and haven't tried reimporting with the seed. Same as Wyssen, because a reset would maybe also lose the current state?

So TLDR: Same Issue as above, advice would be greatly appreciated. I can link screenshots if necessary.

Oh yeah, "gettokenbalances" gives me output as well. Although this output is different from the DFI Holdings I am shown when opening the "DEX" Tab and chosing for example DFI - BTC Swap. My DFI Balance is different here.

Thanks in advance. Dave

**Comment by madmehn** [comment]
That´s also my problem !!! i  tryed also to importwallet "filename" with console  , syntax importwallet "C:\Users\UserName\finan*\*\defichain2812txt" .. result is --> null ? please someone from the team can help ?

**Comment by wyssen** [comment]
FYI @madmehn and @jdavidhuebner I did reimport the seed on another computer and everything worked fine. All the wallets have been recreated. The only thing I needed to do additionally was to do "Menu->Wallet->Import Wallet". This was necessary in order to see all the **addresses** I created before. But even without this step and just recovering with the mnemonic seed, the different wallets under the "Wallet"-Tab were visible with the balances (even the pool-shares etc). Once I knew that everything works out fine, I removed the app on my initial laptop, removed the folders (backuped before) under Users\xyz\AppData\Roaming\defi chain et defi... and reinstalled the latest version. Then I did the same here.

What is very important (I don't know yet if this works) is that once you close the app, you wait for another 20-30 minutes before shutting down the machine. Check in the taskmgr if the defi-process still runs. Wait until it closes by itself and only then shutdown the machine. This should solve the issue with the reindexing everytime (accoridng to the telegram chats).

I am sure that the team will fix this issue soon.
I hope this also works for you guys.
Cheers

**Comment by izzycsy** [comment]
Hi @wyssen , thanks for your feedback, glad you manage to temporarily resolve your issue. Yes, the team is currently investigating this issue. I recommend to do wallet backups every now and then, just in case.

**Comment by izzycsy** [comment]
Hi @jdavidhuebner @madmehn , thanks for your feedback, could you send your app logs please? 

<img width="339" alt="1 App log" src="https://user-images.githubusercontent.com/65014479/103275988-54e9dc80-4a00-11eb-8b1e-32250e1752d0.png">

<img width="511" alt="2 Compress" src="https://user-images.githubusercontent.com/65014479/103276039-7d71d680-4a00-11eb-82e2-cbb3e1026452.png">

<img width="671" alt="3 Drag and drop" src="https://user-images.githubusercontent.com/65014479/103276065-89f62f00-4a00-11eb-974d-bab6b7c30d82.png">

**Comment by wyssen** [comment]
Hi @izzycsy . Thank for your reply. Unfortunately, even after the restore via seed and then the import of the wallet, I am not able to see the different wallet addresses. Recovering the seed does show the Coins, but I don't see the addresses. Without the addresses, I am not able to reinvest into LM. Do you have an idea how to solve this issue? Even when I recreat a new address, it is not visible in the dropdown when adding liquidity to LM...

**Comment by izzycsy** [comment]
Hi @wyssen , could you send screenshots of the issue you're facing? Also, kindly attach your app logs please. Thank you!

**Comment by madmehn** [comment]
thank you for your detailed explanation. had hoped it works. have also tried 3 other computers. after the version 2.1.3 and 2.1.4 were installed in each case was syncronized. after reaching 100% I get as a thank you a white screen and can do nothing more. thanks to the incompetent programmers,

**Comment by izzycsy** [comment]
Hi @madmehn , thanks for your feedback, please give us time to investigate this issue.

**Comment by wyssen** [comment]
> Hi @wyssen , could you send screenshots of the issue you're facing? Also, kindly attach your app logs please. Thank you!

I did downgrade to 2.1.3 and was able to import the wallet and see the old labels. However, they have new addresses, but I can see them in the dropdowns know. I wait until the fixed 2.1.5 gets out and try to update then. Until then, I wait with 2.1.3 and don't shut down my computer anymore ;)
Thank you for your fast support @izzycsy and good luck fixing 💯

**Comment by jdavidhuebner** [comment]
Hi Guys. Back from Christmas holidays. Please find my AppLogs attached.

Sadly, I still have the same problem on 2.16. My wallets are gone and I can't select any of the wallet dropdown menu options. "Import Wallet" "BackUp Wallet" and "Reset Wallet" are not available/clickable. I'm still afraid of clicking "Import Wallet via Mnemonic Seed" since I have a lot of farmed DFI currently shown as balance on the DEX. But no wallets to be seen. What do I do?
Import via Mnemonic Seed? Reinstall? Help would greatly be appreciated.

[AppLogs.log](https://github.com/DeFiCh/app/files/5791796/AppLogs.log)

**Comment by jdavidhuebner** [comment]
Any News here @izzycsy ?

**Comment by izzycsy** [comment]
Hi @jdavidhuebner , did you wait for the synchronisation to reach 100%? The current update would be the recent v2.1.6. Could you send a screenshot of what you mean by unable to click on `import wallet` please, it's located at the _top menu bar of your desktop_, not the app. 

I'd suggest to do regular `backup wallet` between each update. In the event that you need to **Restore Wallet** via mnemonic seed and some random bug occurs, you can then `import` backup to get back to where you left off.

**Comment by jdavidhuebner** [comment]
Hi @izzycsy - Thanks for the response. Yes I wait for 100%. I have the current Update. I took a screenshot, sadly you can't see my cursor. The problem is - I can't select any of the dropdown options under "wallet". They are all greyed/whited out. So I can't even import my backups, since I can't select "Import Wallet". Right now I can't do backups anymore either, since, as I said, I can't select anything under the "Wallet"-Dropdown Menu
![Screenshot (3)](https://user-images.githubusercontent.com/7792797/104216926-fd556300-543a-11eb-8018-f4c7039f1bcd.png)

I haven't yet tried to "Restore Wallet with Mnemocid Seed" since I'm afraid my DFI rewards will not be paid out.

**Comment by izzycsy** [comment]
Hi @jdavidhuebner , thanks for the screenshot, please give us time to investigate. Yes, I think it's wise wait at the moment. Thank you for your patience.

**Comment by izzycsy** [comment]
Hi @wyssen , update v2.1.6 is out <https://defichain.com/downloads/>.

Do give a thumbs up to this comment if your issue has been resolved, otherwise, please send new information (include screenshots) and your app logs of the issue you're facing. Thank you.

**Comment by wyssen** [comment]
Yes, it works perfect now, thank you so much. The only thing I was not able to to is to reimport the old addresses. However, I created new ones and everything works great. Super fast. YOU ARE AWESOME, thank you.

**Comment by jdavidhuebner** [comment]
Hi @izzycsy!

Which Buttons do you mean by greyed out? Only all options in the "Wallet" Dropdown Menu are greyed out, everything else works. I even see my DFI Balance going up when I check on available Balance on the Dex.
Here are all the requested Logs from today. I just can't add the _app config_ since it creates a ".conf"-file and Github doesn't let me upload that. I converted it to a .txt-file. Hope that helps.

[Defi App Config 12-01-2021.txt](https://github.com/DeFiCh/app/files/5801669/Defi.App.Config.12-01-2021.txt)

[Defi BinaryLogs 12-01-2021.log](https://github.com/DeFiCh/app/files/5801621/Defi.BinaryLogs.12-01-2021.log)
[Defi AppLogs 12-01-2021.log](https://github.com/DeFiCh/app/files/5801629/Defi.AppLogs.12-01-2021.log)

**Comment by izzycsy** [comment]
Hi @jdavidhuebner , logs well received, please give the team time to investigate this issue. Because your issue `Unable to click Import / Backup / Reset wallet` is new, may I suggest that you **note down your wallet address** for safe keeping, in case you face the another ongoing bug, and cannot `backup wallet`. Thank you.

**Comment by izzycsy** [comment]
Hi @jdavidhuebner , I suggest that you keep your wallet on the screen you're at, do not change anything. It could be like the app "forgot" that you've a wallet, the team is still investigating this. Please open a new Issue for this as mentioned above.
